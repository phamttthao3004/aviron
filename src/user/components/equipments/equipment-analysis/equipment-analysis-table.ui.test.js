import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import EquipmentAnalysisTableUI from './equipment-analysis-table.ui';

describe('Equipment Analysis Table UI', () => {
    it('Render Equipment Analysis Table UI component', () => {
        const props = { selectAll: undefined, stateTable: undefined, onSearch: undefined, onChangeInput: undefined };
        const com = shallow(<EquipmentAnalysisTableUI {...props} />);
        expect(shallowToJson(com)).toMatchSnapshot();
    });
});
