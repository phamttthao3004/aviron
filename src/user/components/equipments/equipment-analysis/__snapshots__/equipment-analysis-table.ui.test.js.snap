// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Equipment Analysis Table UI Render Equipment Analysis Table UI component 1`] = `
<div
  className="equipment-analysis-col-left"
>
  <div
    className="form-inline"
  >
    <div
      className="input-group input-search-container"
    >
      <input
        className="form-control"
        onChange={[Function]}
        placeholder="Search"
        type="text"
      />
      <span
        className="input-group-btn"
      >
        <button
          className="btn btn-default"
          onClick={[Function]}
          type="button"
        >
           
          <span
            aria-hidden="true"
            className="glyphicon glyphicon-search"
          />
        </button>
      </span>
    </div>
  </div>
  <label
    htmlFor="select-all"
  >
    <input
      id="select-all"
      onClick={[Function]}
      type="checkbox"
    />
    <span>
      All
    </span>
  </label>
  <div
    className="cms-table-content"
  >
    <table
      className="table aviron-table action-table call-data-table"
    >
      <tbody />
    </table>
  </div>
</div>
`;
